content/platforms/kubernetes/tasks/ldap-on-k8s.md

```diff
@@ -0,0 +1,501 @@
+---
+Title: Configuring LDAP Authentication
+description: This task describes how to configure LDAP-based authentication for a cluster on Kubernetes.
+weight: 90
+alwaysopen: false
+categories: ["Platforms"]
+aliases:
+---
+
+
+Throughout this tutorial we will assume there is a single namespace called ‘db'.
```

Comment:
```suggestion
This tutorial describes how to configure LDAP-based authentication for a Redis Enterprise cluster on Kubernetes. Throughout this tutorial, we will assume that there is a single namespace called ‘db'.
```

Reply:
@alexmilowski Can you summarize the steps of the tutorial up front? There's a lot of information here, and a little roadmap would be useful.